ArduCopter/Log.cpp

```diff
@@ -547,8 +584,24 @@ const struct LogStructure Copter::log_structure[] = {
 // @Field: aY: Target acceleration, Y-Axis
 // @Field: aZ: Target acceleration, Z-Axis
 
-    { LOG_GUIDEDTARGET_MSG, sizeof(log_GuidedTarget),
-      "GUID",  "QBfffbffffff",    "TimeUS,Type,pX,pY,pZ,Terrain,vX,vY,vZ,aX,aY,aZ", "s-mmm-nnnooo", "F-BBB-BBBBBB" , true },
+    { LOG_GUIDED_POSITION_TARGET_MSG, sizeof(log_Guided_Position_Target),
+      "GPOS",  "QBfffbffffff",    "TimeUS,Type,pX,pY,pZ,Terrain,vX,vY,vZ,aX,aY,aZ", "s-mmm-nnnooo", "F-BBB-BBBBBB" , true },
+
+// @LoggerMessage: GATT
+// @Description: Guided mode attitude target information
+// @Field: TimeUS: Time since system startup
+// @Field: Type: Type of guided mode
+// @Field: Roll: Target attitude, Roll
+// @Field: Pitch: Target attitude, Pitch
+// @Field: Yaw: Target attitude, Yaw
+// @Field: RollRt: Roll rate in centidegrees/s
+// @Field: PitchRt: Pitch rate in centidegrees/s
+// @Field: YawRt: TYaw rate in centidegrees/s
```

Comment:
The unit type should be handled by the logger fields themselves. To make an output in degrees/sec. These comments are actually wrong since with the multiplier they are in deg/sec.

```suggestion
// @Field: RollRt: Roll rate
// @Field: PitchRt: Pitch rate
// @Field: YawRt: TYaw rate
```

Reply:
I would say, convert everything to degrees/s before logging and leave the "in degrees/s"

Reply:
@lthall I left them in centidegrees because in the former GUID message too we were logging values in centimeters, centidegrees etc. So I tried to maintain that consistency. If you still say, I would be happy to convert these :-).

Reply:
I would strongly support moving to degrees. If we are going to change them lets do it properly.

Reply:
Sure! I will change all units to degrees and deg/s wherever required. Thanks for your suggestion.